### ISSUE TITLE
Possible to throw a Failed or Warning Status?

### ISSUE THREAD

**Issue by TinuS-OG** [aelassas/Wexflow#284]
Hello Akram

I do have a workflow manipulating some files of a third party software and moving them forward to a next software to further proccess them.

At the beginning of my workflow I do test, if the files do exist. Only if they do exist I do continue with the process. So far so good, all works well.

I do abort the further processing, if those files don't exist. For that I'd like to use the Flowchart Else Case like:

							<Else>
								<Task id="110">
									<Parent id="-1" />
								</Task>
							</Else>

Is it possible to Throw a Workflow Error or Warning here? What then gets displayed and counted as Failed or Warning of this Workflow in the Dashboard or the History of Wexflow? So far I only have seen Warning Counts of my Workflows, if the did run into some file locks or so. 

In case this question would be a feature request, could I start any task, what you know always produces a Failed Status? Any Idea?

I have not found an answer to my question in the Wexflow description pages, therefore I'm asking.

**Comment by aelassas** [comment]
No, it's not possible to throw a workflow error or warning for the moment. This would be a feature request.

**Comment by TinuS-OG** [comment]
Hi Akram

Ok, all clear. Please do add this story to your feature request list then.

My workaround in the meanwhile I was able to get to work. In my Workflow else case I do delete a non-existing dummy file what then throws an failed status for the whole workflow. And then it appears 'as red' within the dashboard/history and it's easier  for me to locate.

Thank you, greetings, Martin
